docs/config_options.md

```diff
@@ -94,68 +101,207 @@ HTTP_RATE_LIMIT_CONNECTIONS=200 # Max number of requests during window. Defaults
 JSON_RPC_STRICT_MODE=false
 ```
 
-## Lisk settings
+### External dependancies
+
+```bash
+# URL of the volatile cache storage (Redis)
+SERVICE_GATEWAY_REDIS_VOLATILE='redis://localhost:6379/5'
+```
 
-### Node settings
+### Operational settings
 
 ```bash
-LISK_CORE_WS=ws://localhost:8080       # Lisk Core WebSocket RPC API
-LISK_CORE_CLIENT_TIMEOUT=30            # Lisk Core client timeout (in seconds)
+
+# Services on which the gateway is dependent
+# Can be expressed as a CSV
+GATEWAY_DEPENDENCIES='indexer,connector'
+
+# Job run interval/schedule to update the readiness status.
+# By default, interval is set to 0 (disabled) and schedule is set to run every minute (* * * * *).
+# Interval takes priority if it is set to a number greater than 0.
+JOB_INTERVAL_UPDATE_READINESS_STATUS=0
+JOB_SCHEDULE_UPDATE_READINESS_STATUS='* * * * *'
 ```
 
-Deprecated settings
 
+## App Registry
+
+### External dependancies
+```bash
+# Mysql settings
+SERVICE_APP_REGISTRY_MYSQL=mysql://lisk:password@mysql-primary:3306/lisk
+
+# Truncate the table and rebuild at application init. Defaults to false.
+ENABLE_REBUILD_INDEX_AT_INIT=false
+```
+
+### GitHub app registry settings
+```bash
+# GitHub respository and branch
+# GitHub token with read access is required incase the repo is private
+GITHUB_ACCESS_TOKEN=''
+GITHUB_APP_REGISTRY_REPO=''https://github.com/LiskHQ/app-registry
+GITHUB_APP_REGISTRY_REPO_BRANCH='main'
 ```
-# This setting is required only for SDK version 4 or lower.
-LISK_CORE_HTTP=https://mainnet.lisk.com # Lisk Core HTTP URL
+
+### Operational settings
+```bash
+# Default blockchain applications. By default it is set to lisk_mainchain.
+# Can be expressed as csv.
+DEFAULT_APPS=lisk_mainchain
```

Comment:
Apart from the common environment variables, please remove the entries for all other microservices and point to the specific README files. This will help us reduce maintenance tremendously.